Packages/com.unity.inputsystem/CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ however, it has to be formatted properly to pass verification tests.
 
 - Fixed pairing devices to existing `InputUser`s potentially corrupting list of paired devices from other `InputUser`s ([case 1327628](https://issuetracker.unity3d.com/issues/input-system-devices-are-reassigned-to-the-wrong-users-after-adding-a-new-device)).
 - Fixed StackOverflowException caused by calling InputSystem.Update from inside an input action callback ([case 1316000](https://issuetracker.unity3d.com/issues/crash-when-adding-inputsystem-dot-update-to-inputsystem-command-handler-to-force-processing-an-event-and-sending-input)).
+- Fixed setting both the light bar and motor speeds simultaneously on a DualShock 4 controller ([case 1271119](https://fogbugz.unity3d.com/f/cases/1271119)).
```

Comment:
Think we haven't actually really fixed the problem but rather provided a workaround. If you call the two separate methods independently, you still run into the same problem.

Should we move this to `Added` and say something like "Added workaround to ...."?

Reply:
Good call